interfaces/acados_matlab_octave/acados_ocp.m

```
@@ -133,9 +133,13 @@
                 disp('found compiled acados MEX interface')
             end
 
-            % create C object
-            obj.C_ocp = ocp_create(obj.model_struct, obj.opts_struct);
-
+            try
+                % create C object
+                obj.C_ocp = ocp_create(obj.model_struct, obj.opts_struct);
+            catch ex
+                sprintf('Exception:\n\t%s\n\t%s\n',ex.identifier,ex.message);
```

I like the `try`, probably it should return an error on failure, like this:
```
str = sprintf('Exception:\n\t%s\n\t%s\n',ex.identifier,ex.message);
error(str);
```